### DISCUSSION TITLE
test file size

### DISCUSSION THREAD

**Discussion by sbenphia** [CMSgov/price-transparency-guide-validator#54] [Q&A] [answered]
I am planning to download and test the validator next week.  I noticed that there are test files mentioned in the documentation.  How large are these test files - should I make sure I have a lot of space available for the download?  On that same note - if the test files are small, could you please make the larger test files you used when creating the validator available - I am reading in documentation from networks that some files may be extremely large (over 20 gigabytes).  Is there a generated test file that is as large as these files are expected to be so that I can learn the validator in an apples to apples format.  I am also wondering if you wrote something that generates large test files that you could share.

**Comment by shaselton-usds** [comment, marked as answer]
No, the test files aren't in the GB size. You are more than welcome to take some of the test files and make them larger. This can be done pretty easily through VIM or whatever IDE you're working with.
